modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -720,11 +725,16 @@ public String toVarName(String name) {
         // pet_id => petId
         name = camelize(name, true);
 
-        // for reserved word or word starting with number, append _
-        if (isReservedWord(name) || name.matches("^\\d.*")) {
+        // for reserved words surround with ``
+        if (isReservedWord(name)) {
             name = escapeReservedWord(name);
         }
 
+        // for words starting with number, append _
```

Comment:
This is also a breaking change, can you explain the why of this change please?

Reply:
All Swift reserved words can be escaped using backticks as stated [here](https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID412).